antlr4ide.core/src/main/java/com/github/jknack/antlr4ide/console/Console.java

```diff
@@ -0,0 +1,48 @@
+package com.github.jknack.antlr4ide.console;
+
+/**
+ * Console and logger for showing feedback to users.
+ */
+public interface Console {
```

Comment:
Not sure, but should we use the log4j Logger interface combined with AbstractLogger instead of creating our own ?